app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/MultipleAccountChecksController.scala

```diff
@@ -18,21 +18,29 @@ package uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers
 
 import javax.inject.{Inject, Singleton}
 import play.api.Logging
+import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.connectors.IVConnector
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.config.AppConfig
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.connectors.{EACDConnector, IVConnector}
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.auth.AuthAction
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.errors.UnexpectedResponseFromIV
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.errors.{UnexpectedResponseFromEACD, UnexpectedResponseFromIV}
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.UnderConstructionView
 
 import scala.concurrent.{ExecutionContext, Future}
 @Singleton
-class AccountCheckController @Inject()(
+class MultipleAccountChecksController @Inject()(
```

Comment:
It is better to keep this has a AccountCheck and have a new controller for multiple accounts